conda_build_prepare/git_helpers.py

```diff
@@ -290,6 +292,12 @@ def git_checkout(git_repo, revision):
     _call_custom_git_cmd(git_repo, f'checkout {revision}')
 
 
+def git_get_heads_time(git_repo, date_format):
```

Comment:
heads or head?

Reply:
In text I'd say "get HEAD's time" but in this case I guess you're right that `get_head_time` seems better.